packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -568,7 +572,6 @@ class HotRunner extends ResidentRunner {
     // Send timing analytics.
     globals.flutterUsage.sendTiming('hot', 'restart', restartTimer.elapsed);
 
-    // In benchmark mode, make sure all stream notifications have finished.
     if (benchmarkMode) {
```

Comment:
this is for benchmark mode only for, those tests will validate where it stabilizes things